data/sql/updates/pending_db_world/rev_1556311512999196000.sql

```diff
@@ -0,0 +1,5 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1556311512999196000');
+
+UPDATE `quest_template` SET `RewardFactionOverride1`=2500 WHERE `ID`=13662;
+
+UPDATE `quest_template` SET `flags`=1 WHERE `ID`=13662;
```

Comment:
those two `UPDATE` statements can be merged into one:
```sql
UPDATE `quest_template` SET `RewardFactionOverride1`=2500, `flags`=1 WHERE `ID`=13662;
```

Reply:
Done, thanks